### ISSUE TITLE
Inability to read threaded conversation in MS Teams

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#14238]
### Steps to reproduce:
1.Open Microsoft Teams.
2.Navigate to a channel and then on a threaded conversation.
3.Open NVDA.
4.Navigate to the replies of the conversation.
5.Listen the announcement.
### Actual behavior:

NVDA is announcing the replies as menu and no other information is provided about the reply.
### Expected behavior:

NVDA should announce the sender's name and then the content of the message.
#### NVDA version:
2022.3
#### Windows version:
21H2(OS Build 22000.978)
#### Name and version of other software in use when reproducing the issue:
Microsoft Teams:1.5.00.27764(64 bit)
#### Does the issue still occur after restarting your computer?
Yes

**Comment by gauravahir28** [comment]
I don't think that it is an issue.
The threaded reply always gets added at the end.
In threaded conversation, we do not have option to reply to the specific sender.
Also, this shouldn't be a NVDA issue as NVDA is only reading out the label provided to the text field by microsoft which is sufficient according to me.

**Comment by msftedad** [comment]
JAWS and Narrator reads the sender's name and message content properly except NVDA

**Comment by gauravahir28** [comment]
In my case, JAWS also provides same information as NVDA.
didn't try with Narrator though.
but Logically, this shouldn't be an issue as there can be multiple users involved in a threaded conversation.
it is not Necessary that you will always Respond to the last message of the conversation.

**Comment by gauravahir28** [comment]
O! I misunderstood your steps I guess.
Still, NVDA is behaving as expected at my end,
Here is how NVDA announces:
Reply from [Senders name]. Message content.. Sent today at [time]. Press Enter to explore message content, then use Escape to shift focus back to the message.

**Comment by msftedad** [comment]
Can you please tell me your NVDA version?

**Comment by gauravahir28** [comment]
@msftedad - it is same as yours.
NVDA 2022.3
And My MS teams version is  1.5.00.21668 (64-bit).

**Comment by gauravahir28** [comment]
@msftedad  - I updated my MS teams and can replicate this issue now.
The The behavior is quite inconsistent
sometimes, I am able to read through the replys, but most of the time, when focused to the reply of the thread, the behavior is same as mentioned in the issue.

**Comment by michaelDCurran** [comment]
Closing in favor of duplicate issue #14355  as it has clearer steps to reproduce.
